composer.json

```diff
@@ -30,7 +30,7 @@
         "symfony/dom-crawler": ">=2.7 <5.0",
         "behat/gherkin": "^4.4.0",
         "codeception/phpunit-wrapper": "^6.0.9|^7.0.6",
-        "codeception/stub": "^2.0"
+        "codeception/stub": "dev-master"
```

Comment:
This line can be dropped before the merge, but need to stay here for now to prove the goodness of the PR in Travis